jwxml/mirrors.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+import matplotlib.pyplot as plt
+try:
+    from lxml import etree
+    HAVE_LXML = True
+except ImportError:
+    import xml.etree.cElementTree as etree
+    HAVE_LXML = False
+
+import logging
+import unittest
+import os
+_log = logging.getLogger('jwxml')
+
+try:
+    import webbpsf
+    _HAS_WEBBPSF=True
+except ImportError:
+    _HAS_WEBBPSF=False
+
+#---------------------------------------------------------------------------------
+#   Mirror Move related classes
+
+class Segment_Update(object):
+    """ Class for representing one single mirror update (will be inside of groups in SURs)
+    """
+    def __init__(self, xmlnode):
+        if xmlnode.attrib['type'] != 'pose': raise NotImplemented("Only Pose updates supported yet")
+
+        self.id = int(xmlnode.attrib['id'])
+        self.type = xmlnode.attrib['type']
+        self.segment = xmlnode.attrib['seg_id'][0:2]
+        self.absolute = xmlnode.attrib['absolute'] =='true'
+        self.coord= xmlnode.attrib['coord'] #local or global
+        self.stage_type= xmlnode.attrib['stage_type']  # recenter_fine, fine_only, none
+
+        self.units = dict()
+        self.moves = dict()
+        for move in iterchildren(xmlnode):
```

Comment:
I think this would fail right now because the `iterchildren` wrapper is only defined in `siaf.py` and is invisible in `mirrors.py`. Maybe we should stick that in a `util.py`?